src/Symfony/Component/Console/Helper/TableStyle.php

```diff
@@ -132,6 +194,46 @@ public function getCrossingChar()
         return $this->crossingChar;
     }
 
+    public function getCrossingTopRightChar(): string
```

Comment:
Do we need these getters? I can't picture a use case for these.

Reply:
I agree with @ostrolucky, I would remove all the getters.

Reply:
But getters are used in Table class for rendering, they can't be removed.

Reply:
Indded

Reply:
I fear I'll might my the only one, but I'd rather use a single getter to get them at once (from topLeft to midLeft).

Reply:
+1 for @ogizanagi's idea. And let's mark it as internal.

Reply:
Done